cloudera/tests/test_unit.py

```diff
@@ -287,3 +287,149 @@ def test_given_cloudera_check_when_no_events_response_then_no_event_collection(
         )
         # verify that event is not collected, but check still works normally
         aggregator.assert_event(msg_text=expected_content, count=0)
+
+
+def test_given_cloudera_check_when_autodiscover_configured_then_emits_configured_cluster_metrics(
```

Comment:
For this test and also the exclude test, can we assert that the expected tags are included for the metrics? We can do something similar to [this test](https://github.com/DataDog/integrations-core/blob/0bd581981e72826e16ac7048ff457515f9cd68cc/cloudera/tests/test_integration.py#L46-L47) in checking the tags.

Reply:
Very good suggestion. Although after looking at how we are mocking Cloudera's responses in our unit tests, some more in-depth modification would be necessary to make the correct checks for the tags, which I think is a very good idea. Perhaps we can address it in a future PR more thoroughly. What do you think?

Reply:
After talking with Andrew, in a future PR (not high priority) we will review unit tests